Lib/mimetypes.py

```diff
@@ -543,6 +543,10 @@ def _default_mime_types():
         '.webm'   : 'video/webm',
         '.avi'    : 'video/x-msvideo',
         '.movie'  : 'video/x-sgi-movie',
+        '.woff'   : 'application/font-woff',
```

Comment:
The dict is more or less sorted by value. Please move the new 'application' mimetypes up.

Reply:
Hi @tiran, I have sorted it based on values.